pkg/dao/repository_test.go

```diff
@@ -380,6 +380,54 @@ func (suite *ReposSuite) TestListFilterArch() {
 	assert.Equal(t, int64(1), total)
 }
 
+func (suite *ReposSuite) TestListFilterMultipleArch() {
+	t := suite.T()
+	orgID := "4234"
+	pageData := api.PaginationData{
+		Limit:  10,
+		Offset: 0,
+	}
+
+	filterData := api.FilterData{
+		Search:  "",
+		Arch:    "x86_64,s390x",
+		Version: "",
+	}
+
+	err := seeds.SeedRepositoryConfigurations(db.DB, 20, seeds.SeedOptions{OrgID: orgID})
+	assert.Nil(t, err)
+
+	response, count, err := GetRepositoryDao().List(orgID, pageData, filterData)
+
+	assert.Nil(t, err)
+	assert.Equal(t, 10, len(response.Data))
+	assert.Equal(t, int64(14), count)
```

Comment:
I thought there was randomness to the seed data?    I think i'd prefer if you just create 2-3 repository configs explicitly and run the query on those.  That way if seed data changes it doesn't break the test

Reply:
I removed the randomness from the data.